grafana-plugin/src/containers/AddResponders/AddResponders.tsx

```diff
@@ -31,21 +29,24 @@ type Props = {
   generateRemovePreviouslyPagedUserCallback?: (userId: string) => () => Promise<void>;
 };
 
-const LearnMoreAboutNotificationPoliciesLink: React.FC = () => (
-  <a
-    className={cx('learn-more-link')}
-    href="https://grafana.com/docs/oncall/latest/notify/#configure-user-notification-policies"
-    target="_blank"
-    rel="noreferrer"
-  >
-    <Text type="link">
-      <HorizontalGroup spacing="xs">
-        Learn more
-        <Icon name="external-link-alt" />
-      </HorizontalGroup>
-    </Text>
-  </a>
-);
+const LearnMoreAboutNotificationPoliciesLink: React.FC = () => {
+  const styles = useStyles2(getAddRespondersStyles);
+  return (
+    <a
+      className={cx(styles.learnMoreLink)}
```

Comment:
This is just an example but this comment applies to plenty other similar places - why do we use `cx(styles.someClassName)` instead of just `styles.someClassName`? `cx` is handy in 2 situations: when you want to add multiple classes and/or apply some class conditionally. In case we set a single class without any condition it doesn't make much sense. 

How about we get rid of them in such situations and keep only if it brings benefit?